modules.json

```diff
@@ -0,0 +1,6 @@
+[
+    "pixfutureBidAdapter",
+    "appnexusBidAdapter",
+    "consentManagement",
+    "gdprEnforcement"
+]
```

Comment:
This file is for testing and should be removed from pr

Reply:
Hi Chris,

We removed modules.json and added info to https://github.com/prebid/prebid.github.io
Please let us know about our adapter status.

Regards,
Vitali I.